### ISSUE TITLE
set gyro_lowpass_type = PT1 shows as BIQUAD in configurator

### ISSUE THREAD

**Issue by 0crap** [betaflight/betaflight-configurator#1351]
When I  do "set gyro_lowpass_type = PT1", save my changes and connect again the Configurator, it shows "Gyro Lowpass 1 Dynamic Filter Type" as BIQUAD. (Previously it showed PT1 as expected.)

If I set it, using the GUI to PT1, save and refresh, it's again back at BIQUAD. (Should be PT1.)

My guess is this bug might be related to PR #1339 ?

Running the Jenkins Chrome version with file dates from 4 april.

**Comment by 0crap** [comment]
![BIQUAD](https://user-images.githubusercontent.com/31951195/55756412-c413e600-5a51-11e9-9ff1-82845eade62c.jpg)
![PT1](https://user-images.githubusercontent.com/31951195/55756417-c6764000-5a51-11e9-9bed-1d7fca4c23f7.jpg)

**Comment by mikeller** [comment]
@0crap: This should have been fixed with the changes in #1348, please try again with the latest configurator from `master`.

**Comment by McGiverGim** [comment]
Yes, the static is disabled when the dynamic is enabled. No matter what value the static has.

**Comment by 0crap** [comment]
Thx for confirming, I wasn't sure if this was correct.
